app/Http/Controllers/StudentController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ilija.savic
+ * Date: 10/2/2017
+ * Time: 2:21 PM
+ */
+
+namespace App\Http\Controllers;
+
+use App\Student;
+use Illuminate\Http\Request;
+use Laravel\Lumen\Routing\Controller;
+
+class StudentController extends Controller
+{
+    /**
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function index(){
+        $student = Student::all();
+
+        return response()->json($student);
```

Comment:
instead of this built-in response try to figure out a way to return a JSON formatted response that is going to look the same all across the application, no matter if success or fail of the request